### PR TITLE
approx number of underrepl/missing records was calculated incorrectly

### PR DESCRIPTION
**Pull request by chaudum**
which lead to different results when viewing on admin ui on different nodes

### CODE REVIEW

File: app/scripts/services/tableinfo.js [hunk 1]
```diff
@@ -2,90 +2,115 @@
...
+      var numUnderreplicatedRecords = (function(){
+        var avgDocsPerPrimaryShard = primaryShards.reduce(function(memo, shard, idx, arr){
+          return memo + shard.avg_docs / arr.length;
+        }, 0);
+        return avgDocsPerPrimaryShard * numUnderreplicatedShards;
+      }());
+
+      var numUnavailableRecords = (function() {
+        var avgDocsPerStartedShard = startedShards.reduce(function(memo, shard, idx, arr){
+          return memo + shard.avg_docs / arr.length;
```
Comment (chaudum) [inline app/scripts/services/tableinfo.js]: this is the actual fix

File: app/scripts/services/tableinfo.js [hunk 2]
```diff
@@ -2,90 +2,115 @@
...
+        return shards.filter(function(shard, idx) {
+          return shard.state == 'UNASSIGNED';
+        }).reduce(function(memo, shard, idx) {
+          return memo + shard.count;
+        }, 0);
+      }());
+
+      var numUnderreplicatedRecords = (function(){
+        var avgDocsPerPrimaryShard = primaryShards.reduce(function(memo, shard, idx, arr){
+          return memo + shard.avg_docs / arr.length;
```
Comment (chaudum) [inline app/scripts/services/tableinfo.js]: this is the actual fix

**Comment by mikethebeer** [comment]
ack

**Comment by mikethebeer** [comment]
ack

**Pull request merged**